packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx

```diff
@@ -0,0 +1,277 @@
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  fontWeight: 400,
+  fontStyle: isItalic ? 'italic' : 'normal'
+}))
+
+const Plan = styled('div')<{tier: TierEnum}>(({tier}) => ({
+  background:
+    tier === 'starter' ? PALETTE.STARTER : tier === 'team' ? PALETTE.TEAM : PALETTE.ENTERPRISE,
+  fontSize: 12,
+  fontWeight: 600,
+  lineHeight: '16px',
+  textTransform: 'capitalize',
+  textAlign: 'center',
+  display: 'flex',
+  flex: 1,
+  margin: '0 8px',
+  flexWrap: 'wrap',
+  justifyContent: 'flex-start',
+  padding: '16px 8px',
+  height: 400,
+  borderRadius: 4,
+  border: `2px solid transparent`,
+  '&:hover': {
+    border: `2px solid ${
+      tier === 'starter'
+        ? PALETTE.GRAPE_500
+        : tier === 'team'
+        ? PALETTE.AQUA_400
+        : PALETTE.TOMATO_200
+    }`
+  }
+}))
+
+const UL = styled('ul')({
+  margin: 0
+})
+
+const LI = styled('li')({
+  fontSize: 18,
+  lineHeight: '32px',
+  color: PALETTE.SLATE_900,
+  textTransform: 'none',
+  fontWeight: 400,
+  textAlign: 'left'
+})
+
+const StyledIcon = styled('div')({
+  height: 24,
+  width: 24,
+  color: PALETTE.SLATE_600,
+  paddingLeft: 8,
+  '&:hover': {
+    cursor: 'pointer'
+  }
+})
+
+const Content = styled('div')({})
+
+const ButtonBlock = styled('div')({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'center',
+  position: 'relative'
+})
+
+const UpgradeButton = styled(FlatPrimaryButton)<{
+  buttonStyle: 'disabled' | 'primary' | 'secondary'
+}>(({buttonStyle}) => ({
+  width: '80%',
+  boxShadow: buttonStyle === 'primary' ? Elevation.Z8 : Elevation.Z0,
+  position: 'absolute',
+  bottom: 0,
+  background:
+    buttonStyle === 'primary'
+      ? PALETTE.GRADIENT_TOMATO_600_ROSE_500
+      : buttonStyle === 'secondary'
+      ? PALETTE.WHITE
+      : PALETTE.SLATE_300,
+  color:
+    buttonStyle === 'primary'
+      ? PALETTE.WHITE
+      : buttonStyle === 'secondary'
+      ? PALETTE.SLATE_900
+      : PALETTE.SLATE_600,
+  border: buttonStyle === 'secondary' ? `1px solid ${PALETTE.SLATE_600}` : 'none',
+  ':hover': {
+    background:
+      buttonStyle === 'primary'
+        ? PALETTE.GRADIENT_TOMATO_700_ROSE_600
+        : buttonStyle === 'secondary'
+        ? PALETTE.SLATE_100
+        : PALETTE.SLATE_300
+  }
+}))
+
+const OrgPlans = () => {
+  const stats = [
+    {
+      label: 'Active Teams',
+      value: 18
+    },
+    {
+      label: 'Active Members',
+      value: 18
+    },
+    {
+      label: 'Total Meetings',
+      value: 18
+    }
+  ]
+
+  const plans = [
+    {
+      tier: 'starter',
+      subtitle: 'Free',
+      stats: [
+        '2 teams',
+        'Essential templates',
+        'Retrospectives, Sprint Poker, Standups, Check-Ins',
+        'Unlimited team members'
+      ],
+      buttonStyle: 'disabled',
+      buttonLabel: 'Current Plan'
+    },
+    {
+      tier: 'team',
+      stats: ['Everything in Starter', 'Premium templates', 'Custom templates', 'Unlimited teams'],
```

Comment:
In the designs, it says `Unlimited team members`, but I've changed that to `Unlimited teams`